sdk/eventhubs/azure-messaging-eventhubs-track1-perf/src/main/java/com/azure/messaging/eventhubs/perf/EventProcessorOptions.java

```diff
@@ -22,6 +22,9 @@
     @Parameter(names = {"-e", "--eventsToSend"}, description = "Number of events to send per partition.")
     private int eventsToSend = 100000;
 
+    @Parameter(names = {"--publish"}, description = "Switch to indicate whether to publish messages or not.")
```

Comment:
Is it to test receive-then-send scenario? I don't see this option in track2.

Reply:
More so because it takes forever to publish events that sit in the Event Hub for x days. We always start from the beginning of the stream.